app/src/main/java/org/p2p/wallet/push_notifications/model/DeviceToken.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.push_notifications.model
+
+import com.google.gson.annotations.SerializedName
+
+data class DeviceToken(
```

Comment:
DeviceTokenRequest

Reply:
It's not request, it's domain model

Reply:
why it's serialized then?

Reply:
We've got this from the NotificationService, so it should be parsed

Reply:
so then it's DeviceTokenResponse